packages/repository/src/query.ts

```diff
@@ -480,13 +472,14 @@ export class FilterBuilder<MT extends object = AnyObject> {
     if (!this.filter.fields) {
       this.filter.fields = {};
     }
+    const _fields = this.filter.fields;
```

Comment:
Why underscore? Can we use `fields` or perhaps `allFields`/`filterFields` to better distinguish from `field` name used later?